src/cargo/core/compiler/custom_build.rs

```diff
@@ -1,3 +1,36 @@
+//! A module about how to execute a custom build script and parse its output.
```

Comment:
nit: "A module about" is a bit redundant, rustdoc makes it clear this is a module